libs/s25main/ingameWindows/IngameWindow.h

```diff
@@ -99,24 +110,22 @@ class IngameWindow : public Window
     std::string title_;
     glArchivItem_Bitmap* background;
     DrawPoint lastMousePos;
-    bool last_down;
-    bool last_down2;
-    std::array<ButtonState, 2> buttonState;
 
     /// Offset from left and top to actual content
     Extent contentOffset;
     /// Offset from content to right and bottom boundary
     Extent contentOffsetEnd;
 
-    /// Get bounds of close button (left)
-    Rect GetCloseButtonBounds() const;
-    /// Get bounds of minimize button (right)
-    Rect GetMinimizeButtonBounds() const;
-
 private:
+    static constexpr auto ButtonSize = Extent(16, 16);
```

Comment:
I can move this into the source and into an anonymous namespace. Absent C++17's `static inline constexpr`, this is also more convenient.

Reply:
Yes, annoying that you need a definition for a constexpr variable -.-
But doesn't matter much, for me it's fine here too

Reply:
Done.